bindings/go/constructor/construct.go

```diff
@@ -29,20 +29,25 @@ import (
 // e.g. because the component version already exists in the target repository.
 var ErrShouldSkipConstruction = errors.New("should skip construction")
 
+const AttributeDescriptor string = "constructor/descriptor"
```

Comment:
is there a particular reason we use attribute descriptor instead of the value attribute from dag?